Makefile

```diff
@@ -11,10 +11,12 @@ DOCKER_COMPOSE ?= $(DOCKER) compose -f docker-compose.yml $(NODE_COMPOSE_ARGS)
 DOCKER_BUF := $(DOCKER) run --rm -v $(CURDIR):/workspace --workdir /workspace bufbuild/buf
 GOFLAGS := ""
 GOPATH ?= '$(HOME)/go'
-OLD_VERSION := v36.0.1
+OLD_ZETACORED_VERSION := v36.0.1
+OLD_ZETACLIENTD_VERSION := zetaclient_v37.0.3
+OLD_ZETAE2E_VERSION := $(OLD_ZETACORED_VERSION)
```

Comment:
This is not being used currently for zetaclient upgrade tests, as we are not setting  USE_ZETAE2E_ANTE to true 



Correctly validating the second run 
https://github.com/zeta-chain/zeta-node/blob/dbcb3f7050e2ac029fc5353ad8304d3c77ea0f01/e2e/runner/upgrade.go#L129-L135
would allow us to 

OLD_ZETAE2E_VERSION := $(OLD_ZETACLIENTD_VERSION) and then set USE_ZETAE2E_ANTE = true 

We can enable this back once a zetaclient release is created using the updated code 


But this is fine for now as the zetae2e from version v37.0.3 is same as the zetae2e in the current branch